### ISSUE TITLE
puppeteer version always "UNKNOWN"

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2795]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.42.5](https://github.com/open-wa/wa-automate-nodejs/compare/4.42.4...4.42.5) (2022-07-14)

- Release 4.42.5 [`9ed718a`](https://github.com/open-wa/wa-automate-nodejs/commit/9ed718a1d17ceec4386b01dfb7919b9e328b2cc3)
- 🐛 fix puppeteer `UNKNOWN` issue #2795 [`448a595`](https://github.com/open-wa/wa-automate-nodejs/commit/448a5951944154fcbae585cb53842028f8b1bf40)
- 🥚 patches update [`bd6b4e2`](https://github.com/open-wa/wa-automate-nodejs/commit/bd6b4e27f6810cb66b80d29fc5d26f896276114e)
- updated types-only package [`3f6cdaa`](https://github.com/open-wa/wa-automate-nodejs/commit/3f6cdaabd1f295fe835a336cd7dd369db9f2a96a)
